text/0117-unbrick-collective.md

```diff
@@ -0,0 +1,184 @@
+stops producing blocks.
+
+1. **Opt-in to the Unbrick Collective**: This is done by delegating the handling of the para
+  in the _paras registrar_ to an origin related to the Collective. This doesn't require unlocking
+  the para. This way, the collective is enabled to perform changes in the _paras_ module, after
+  the **Unbrick Process** proceeds.
+2. **Request a Para Unlock**: In case the para hasn't delegated its handling in the _paras
+  registrar_, it'll be still possible for the para team to submit a proposal to unlock the para,
+  which can be assisted by the Collective. However, this involves submitting a proposal to the `Root`
+  governance origin.
+
+### Belonging to the Collective
+
+The collective will be initially created without members (no seeding). There will be additional
+governance proposals to setup the seed members.
+
+The origins able to modify the members of the collective are:
+
+- The `Fellows` track in the Polkadot Fellowship.
+- `Root` track in the Relay.
+- More than two thrids of the existing Unbrick Collective.
+
+The members are responsible to verify the technical details of the unbrick requests (i.e. the hash
+of the new PVF being set). Therefore, they must have the technical capacity to perform such tasks.
+
+Suggested requirements to become a member are the following:
+
+- Rank 3 or above in the Polkadot Fellowship.
+- Being a CTO or Technical Lead in a para team that has opted-in to delegate the Unbrick Collective
+  to manage the PVF/head state of the para.
+
+## Drawbacks
+
+The ability to modify the Head State and/or the PVF of a para means a possibility to perform
+arbitrary modifications of it (i.e. take control the native parachain token or any bridged assets
+in the para).
+
+This could introduce a new attack vectorm, and therefore, such great power needs to be handled
+carefully.
+
+## Testing, Security, and Privacy
+
+The implementation of this RFC will be tested on testnets (Rococo and Westend) first.
+
+An audit will be required to ensure the implementation doesn't introduce unwanted side effects.
+
+There are no privacy related concerns.
+
+## Performance, Ergonomics, and Compatibility
+
+### Performance
+
+This RFC should not introduce any performance impact.
+
+### Ergonomics
+
+This RFC should improve the experience for new and existing parachain teams, lowering the barrier
+to unbrick a stalled para.
+
+### Compatibility
+
+This RFC is fully compatible with existing interfaces.
+
+## Prior Art and References
+
+- [RFC-0014: Improve Locking Mechanisms for Parachains][RFC-0014]
+- [How to Recover a Parachain, Polkadot Forum][forum:673]
+- [Unbrick Collective, Polkadot Forum][forum:6931]
+
+## Unresolved Questions
+
+- What are the parameters for the `WhitelistedUnbrickCaller` track?
+- Any other methods that shall be updated to accept `Unbrick` origin?
+- Any other requirements to become a member?
+- We would like to keep this simple, so no funding support from the Polkadot treasury. But do we
```

Comment:
IMO most of the members should be from different parachain teams. This way it is a quid-pro-quo in longterm.

Reply:
But depending on their involvement they could maybe get retroactive payment for helping with a certain fix.